content/documentation/pages/7-resources/3-faq.md

```diff
@@ -417,6 +417,40 @@ See the [compatibility matrix](https://github.com/spring-cloud/spring-cloud-stre
 
 <!--END_QUESTION-->
 
+<!--QUESTION#lifecycle-->
+
+Can I manage binding lifecycles?
+
+By default, bindings are started automatically when the application is initialized.
+Bindings implement the Spring `SmartLifecycle` interface.
+`SmartLifecycle` allows beans to be started in phases.
+Producer bindings are started in an early phase (`Integer.MIN_VALUE + 1000`).
+Consumer bindings are started in a late phase (`Integer.MAX_VALUE - 1000`).
```

Comment:
I wonder if we need to mention `Integer.MIN_VALUE + 1000`. Could be distracting `early/late` is more then enough IMHO.

Reply:
Well, we need to tell them how they can ensure their bean(s) can be started before producers or after consumers; so if we just say "early" and "late" we'd then need to say something like.

>To start a bean before the producer binding, it must be in phase less than `Integer.MIN_VALUE - 1000` 

(and a similar sentence for starting beans after consumer bindings.

So, I am not sure what the benefit would be.

Reply:
I agree with that, but that would be rather very advance concept and in the spirit of _'less is more'_  I'd rather know why a user wants to start/stop something deliberately before/after etc. . .

I am good either way, just throwing it out there to see if any one else has an opinion about that.